.gitignore

```diff
@@ -0,0 +1,42 @@
+# See https://docs.github.com/get-started/getting-started-with-git/ignoring-files for more about ignoring files.
```

Comment:
.vscode directory would be added to .gitignore file

Reply:
It's already in the auto-generated template.